game_api.txt

```diff
@@ -395,3 +395,31 @@ default.grow_new_pine_tree(pos)
 
 default.grow_new_acacia_tree(pos)
 ^ Grows a new design acacia tree at pos
+
+#Tnt API
+----------
+tnt.register_tnt(definition) Register a new type of tnt.
+
+definition
+{
+	name = "tnt"  --The name of the node. If no perfix is give `tnt` is used.
+	description = "TNT"  --A description for your tnt.
+	radius = 3  --The radius in which the tnt can damage nodes. The radius by default is 3	
+	damage_radius = 6  --The radius within which the tnt can damage players and mobs. By default it is twice the `radius`.
+	disable_drops = false  --Disable drops. By default it is set to false
+	tiles={
+	side = "" --side tiles. By default the name of the tnt with a suffix of _side.png
+	top = "" --top tile. By default the name of the tnt with a suffix of _top.png
+	bottom = "" --bottom tile. By default the name of the tnt with a suffix of _bottom.png
+	burning = "" --top tile when lit. By default the name of the tnt with a suffix of _top_burning_animated.png
+	
+	},
+	
+}
+
+tnt.boom(position, radius, damage_radius, disable_drops) 
+^ Create an explosion.
+*position the centrer of explosion
+*`radius` the distance form centrer that block can get damaged in.
+*`damage_radius` the distance form centrer that players and mobs can get damaged in.
+*`disable_drops`  Disable drops.
```

Comment:
I'd prefer if this wasn't a negative. Eg: "drop_nodes" then replace "disable_drops" with "not drop_nodes"